### DISCUSSION TITLE
Distinction between motifSet and motifPWMs

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1502] [Questions / Documentation] [answered]
If I were to pull in different motifPWMs databases when running addMotifAnnotations() what should I enter for the motifSet?

For instance if I include homer_pwms, mouse_pwms and encode_pwms what are my options for motifSet= " ? "

Thanks!

**Comment by rcorces** [comment, marked as answer]
`motifSet` refers to a pre-defined motif set curated in ArchR. `motifPWMs` refers to a user-supplied `PWMatrixList` object.
These two objects are mutually exclusive. You only provide an argument to one of them.
https://www.archrproject.com/reference/addMotifAnnotations.html

**Reply by rcorces** [reply]
docs updated to make this more clear via https://github.com/GreenleafLab/ArchR/commit/39115e45837351f38493353060b32851474c5333

**Reply by sylestiel** [reply]
Thank you!
